dev-client/src/screens/SlopeScreen/utils/renderValues.ts

```diff
@@ -23,13 +23,38 @@ export const renderSteepness = (
   {slopeSteepnessDegree, slopeSteepnessPercent, slopeSteepnessSelect}: SoilData,
 ) =>
   slopeSteepnessSelect
-    ? t(`slope.steepness.select_labels.${slopeSteepnessSelect}`)
+    ? renderSlopeSteepnessSelectInline(t, slopeSteepnessSelect)
     : typeof slopeSteepnessPercent === 'number'
       ? `${slopeSteepnessPercent.toFixed(0)}%`
       : typeof slopeSteepnessDegree === 'number'
         ? `${slopeSteepnessDegree}°`
         : undefined;
 
+export const renderSteepnessForNarrowDisplay = (
+  t: TFunction,
+  {slopeSteepnessDegree, slopeSteepnessPercent, slopeSteepnessSelect}: SoilData,
+) =>
+  slopeSteepnessSelect
+    ? t(`slope.steepness.select_labels.${slopeSteepnessSelect}`) +
+      '\n' +
+      t(`slope.steepness.select_labels.${slopeSteepnessSelect}_PERCENT`)
+    : typeof slopeSteepnessPercent === 'number'
+      ? `${slopeSteepnessPercent.toFixed(0)}%`
+      : typeof slopeSteepnessDegree === 'number'
+        ? `${slopeSteepnessDegree}°`
+        : undefined;
+
+export const renderSlopeSteepnessSelectInline = (
+  t: TFunction,
+  slopeSteepnessSelect: string,
+) => {
+  return (
+    t(`slope.steepness.select_labels.${slopeSteepnessSelect}`) +
+    ' ' +
+    t(`slope.steepness.select_labels.${slopeSteepnessSelect}_PERCENT`)
+  );
```

Comment:
question: what does this look like with real data? Seem likely it should be localized (`{s1} {s2}`).

Reply:
It will look like how we display it on the slope steepness screen already: "Moderately Steep 50-60%"
    ![Screenshot 2024-05-21 at 4 02 33 PM](https://github.com/techmatters/terraso-mobile-client/assets/5590815/9f4feac1-0f9c-44d7-86c4-326218a1d80f)

See the changes in `en.json` and lmk if it's still not clear.

Reply:
Please add something to en.json along similar to "{steepness_description} {steepness_amount}" with the space in it. We want to avoid string concatenation like this in the code. (It will break for RTL languages.)